UKHO.ADM/Objects/Objects.ps1

```diff
@@ -267,8 +277,8 @@ class ADChanges {
             $outputString = "$($user.UPN) does not exist, will not be created so cannot be added to $($group.DistinguishedName)"
             if ($this.StringContent.ContainsKey($outputString) -eq $false) {
                 $this.StringContent.Add($outputString, $true)
-                Write-Colour "x Failed to add user $($user.UPN) to group $($group.Name) on $($group.Domain.DistinguishedName):" -Color Magenta
-                Write-Colour -StartTab 1 "$outputString" -Color Magenta
+                Write-Colour "x Failed to add user $($user.UPN) to group $($group.Name) on $($group.Domain.DistinguishedName):" -Color $this.SystemColours.error
+                Write-Colour -StartTab 1 "$outputString" -Color $this.SystemColours.error
```

Comment:
Slight inconsistency I think (not 100% sure. A little bit hazy, use your best judgement). Should `SystemColours.info` be used instead of `SystemColours.error` for this line?

```powershell
Write-Colour "x Failed to add user $($user.UPN) to group $($group.Name) on $($group.Domain.DistinguishedName):" -Color $this.SystemColours.error
Write-Colour -StartTab 1 "$outputString" -Color $this.SystemColours.error
```

Whereas on ln269(and others)

```powershell
Write-Colour "x Failed to add user $($user.SamAccountName) to group $($group.DistinguishedName):" -Color $this.SystemColours.error
Write-Colour -StartTab 1 "$_" -Color $this.SystemColours.info #Echos out the exceptions message
```

Reply:
yes, I think I did a find and replace, missed that double up

Reply:
fixed in latest